Build fails with `Unknown authenticationOk message typeMessage { name: 'authenticationOk', length: 23 }`

### Reproduction steps

1. `make migrate check`


### Expected Results

The build don't fail.

### Actual Results

Logs from https://ci.jenkins.io/job/Infra/job/uplink/job/master/1/console:
```
 The NODE_ENV variable is not set. Defaulting to a blank string.
 The DEBUG variable is not set. Defaulting to a blank string.
 The SENTRY_DSN variable is not set. Defaulting to a blank string.
 Creating network "infrauplinkmaster_default" with the default driver
 Pulling db (postgres:alpine)...
 alpine: Pulling from library/postgres
 Digest: sha256:acf5271bbecd4b8733f4e93959a8d2b536a57aeee6cc4b6a71890aaf646425b8
 Status: Downloaded newer image for postgres:alpine
 Creating infrauplinkmaster_db_1 ... 
 �[1A�[2K
Creating infrauplinkmaster_db_1 ... �[32mdone�[0m
�[1B>> waiting a moment to make sure the database comes online..
 ./tools/docker-compose run --rm node \
 	/usr/local/bin/node /home/jenkins/agent/workspace/Infra_uplink_master/node_modules/sequelize-cli/lib/sequelize db:migrate && \
 ./tools/docker-compose run --rm node \
 	/usr/local/bin/node /home/jenkins/agent/workspace/Infra_uplink_master/node_modules/sequelize-cli/lib/sequelize db:seed:all
 The NODE_ENV variable is not set. Defaulting to a blank string.
 The DEBUG variable is not set. Defaulting to a blank string.
 The SENTRY_DSN variable is not set. Defaulting to a blank string.
 Starting infrauplinkmaster_db_1 ... 
 �[1A�[2K
Starting infrauplinkmaster_db_1 ... �[32mdone�[0m
�[1B
 �[4mSequelize CLI [Node: 10.24.1, CLI: 4.1.1, ORM: 4.38.0]�[24m
 
 Loaded configuration file "config/database.js".
 Using environment "development".
 Fri, 27 Oct 2023 09:54:41 GMT sequelize deprecated String based operators are now deprecated. Please use Symbol based operators for better security, read more at http://docs.sequelizejs.com/manual/tutorial/querying.html#operators at node_modules/sequelize/lib/sequelize.js:242:13
 /home/jenkins/agent/workspace/Infra_uplink_master/node_modules/pg/lib/connection.js:441
   throw new Error('Unknown authenticationOk message type' + util.inspect(msg))
   ^
 
 Error: Unknown authenticationOk message typeMessage { name: 'authenticationOk', length: 23 }
     at Connection.parseR (/home/jenkins/agent/workspace/Infra_uplink_master/node_modules/pg/lib/connection.js:441:9)
     at Connection.parseMessage (/home/jenkins/agent/workspace/Infra_uplink_master/node_modules/pg/lib/connection.js:357:19)
     at Socket.<anonymous> (/home/jenkins/agent/workspace/Infra_uplink_master/node_modules/pg/lib/connection.js:119:22)
     at Socket.emit (events.js:198:13)
     at addChunk (_stream_readable.js:288:12)
     at readableAddChunk (_stream_readable.js:269:11)
     at Socket.Readable.push (_stream_readable.js:224:10)
     at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
 make: *** [Makefile:62: migrate] Error 1
```

### Anything else?

Related: https://github.com/jenkins-infra/helpdesk/issues/3619#issuecomment-1782627994

--- a/package.json
+++ b/package.json
@@ -57,7 +57,7 @@
     "helmet": "^3.13.0",
     "jsonstream": "^1.0.3",
     "passport-github": "^1.1.0",
-    "pg": "^7.4.3",
+    "pg": "^8.7.1",
     "pg-query-stream": "^1.1.2",
     "pug": "^2.0.4",
     "sequelize": "^4.38.0",

